lib/modules/dosomething/dosomething_reportback/includes/ReportbackItem.php

```diff
@@ -76,8 +76,9 @@ public static function find(array $filters = []) {
     }
 
     foreach($results as $item) {
-      $reportbackItem = new static;
-      $reportbackItem->build($item);
+      $load_user = dosomething_helpers_isset($filters['load_user'], NULL, FALSE);
+      $reportbackItem = new static(['ignore' => true]);
```

Comment:
Didn't notice this before. No need for passing the `ignore` to the `new static()`. That's only needed for the `Reportback` class.